docs/architecture/deployment-execution/import.md

```diff
@@ -0,0 +1,184 @@
+(importing-resources)=
+# Importing resources
+
+There are a number of scenarios in which it is necessary to import existing
+resources for management by Pulumi:
+
+* Migrating manually managed resources to an infrastructure-as-code solution;
+* Migrating from other infrastructure-as-code platforms to Pulumi;
+* Moving resources between Pulumi stacks in cases where e.g. `pulumi state move`
+  is not sufficient.
+
+Pulumi offers two approaches for importing resources: the `import` resource
```

Comment:
Could be a follow up but should we also add docs about .get methods? They are not quite imports but very related.